src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java

```diff
@@ -160,4 +163,75 @@ public boolean isHearingFormGenerated() {
         return !caseData.getHearingDocuments().isEmpty();
     }
 
+    @Override
+    public boolean sdoBeenDrawn() {
+        return false;
+    }
+
+    @Override
+    public boolean beforeHearing() {
+        return caseData.getHearingDate().isAfter(LocalDateTime.now().toLocalDate());
+    }
+
+    @Override
+    public boolean isMoreDetailsRequired() {
+        return sdoBeenDrawn() && beforeHearing();
+    }
+
+    @Override
+    public boolean isMediationSuccessful() {
+        return !sdoBeenDrawn()
+            && caseData.getMediation().getMediationSuccessful() != null;
+    }
+
+    @Override
+    public boolean isMediationUnsuccessful() {
+        return !sdoBeenDrawn()
+            && !caseData.getMediation().getUnsuccessfulMediationReason().isEmpty();
+    }
+
+    @Override
+    public boolean isMediationPending() {
+        return caseData.getMediation().getMediationSuccessful() != null
+            && caseData.getCcdState().equals(CaseState.IN_MEDIATION)
+            && caseData.getMediation().getMediationSuccessful().getMediationAgreement() == null;
+    }
+
+    @Override
+    public boolean isCourtReviewing() {
+        return !sdoBeenDrawn()
+            && caseData.getApplicant1ProceedsWithClaimSpec().equals(YesOrNo.YES)
+            && caseData.isRespondentResponseFullDefence()
+            && caseData.hasApplicantRejectedRepaymentPlan();
+    }
+
+    @Override
+    public boolean isClaimEnded() {
+        return caseData.getApplicant1ProceedsWithClaimSpec().equals(YesOrNo.NO)
+            && caseData.isRespondentResponseFullDefence();
+    }
+
+    @Override
+    public boolean claimSentToClaimant() {
+        return caseData.getRespondent1ClaimResponseType().equals(RespondentResponseType.FULL_DEFENCE)
```

Comment:
can we use a method as isRespondentResponseFullDefence() like we used in line 204?

Reply:
sure, fixed